### PR TITLE
Automatically port Reflection.Metadata.Ecma335 N-S APIs

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners: @tmat, @nguerrera

### CODE REVIEW

File: xml/System.Reflection.Metadata.Ecma335/SignatureDecoder`2.xml [hunk 1]
```diff
@@ -186,10 +195,11 @@
         <Parameter Name="allowTypeSpecifications" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="blobReader">To be added.</param>
-        <param name="allowTypeSpecifications">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="blobReader">The blob reader positioned at the leading SignatureTypeCode</param>
```
Comment (rpetrusha) [inline xml/System.Reflection.Metadata.Ecma335/SignatureDecoder`2.xml]: Is there a cref for SignatureTypeCode. Or a link we can add to the remarks section?

Comment (carlossanlop) [reply xml/System.Reflection.Metadata.Ecma335/SignatureDecoder`2.xml]: Yes (found [here](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Reflection.Metadata/SignatureTypeCode.xml)):
```suggestion
        <param name="blobReader">The blob reader positioned at the leading <see cref="T:System.Reflection.Metadata.SignatureTypeCode" />.</param>
```

**Comment by carlossanlop** [comment]
@rpetrusha thanks for making the additional changes, they look good.
Unless @tmat or @nguerrera have any additional remarks, I think we are good to merge, as soon as the build passes with no warnings.

**Comment by rpetrusha** [comment]
There was a broken link in the build report, @carlossanlop, so I've fixed it. We can merge when the build completes successfully.

**Pull request merged**